test/sql.test.js

```diff
@@ -84,7 +84,7 @@ test('shorthand for node-postgres', async () => {
       if (text === 'select * from books') {
         return { rows: sampleBooks }
       }
-      return { rows: false }
+      return { rows: [] }
```

Comment:
According to `pg`'s declaration file, the `rows` field of a query result (`QueryResult`) is always an array.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f0af8db482f3b7ca6b01d11f84fd7cda5be2e9a9/types/pg/index.d.ts#L80